packages/forklift-console-plugin/src/modules/Providers/views/details/tabs/VirtualMachines/components/MigrationAction.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { FC } from 'react';
+import { useHistory } from 'react-router';
+import { getResourceUrl } from 'src/modules/Providers/utils';
+import { useCreateVmMigrationData } from 'src/modules/Providers/views/migrate';
+import { useForkliftTranslation } from 'src/utils/i18n';
+
+import { PlanModelRef, V1beta1Provider } from '@kubev2v/types';
+import { Button, ToolbarItem } from '@patternfly/react-core';
+
+import { VmData } from './VMCellProps';
+
+export const MigrationAction: FC<{
+  selectedVms: VmData[];
+  provider: V1beta1Provider;
+}> = ({ selectedVms, provider }) => {
+  const { t } = useForkliftTranslation();
+  const history = useHistory();
+  const namespace = provider?.metadata?.namespace;
+  const planListURL = getResourceUrl({
+    reference: PlanModelRef,
+    namespace,
+    namespaced: namespace !== undefined,
+  });
+  const { setData } = useCreateVmMigrationData();
+  return (
+    <ToolbarItem>
+      <Button
+        variant="secondary"
+        onClick={() => {
+          setData({ selectedVms, provider });
+          history.push(`${planListURL}/~new`);
+        }}
+        isDisabled={!selectedVms?.length}
+      >
+        {t('Migrate')}
```

Comment:
the buttons are
"create and start" -> create plan + migration and then move you to "details" page where you can see the progress
"create and edit" -> create plan and then move you do "details" page for editing the new plan or just start it manually
"cancel" -> return to last page, in this flow the providers "details" page on "vm" tab

Reply:
ahh , sorry, wrong button